toolchain/parser/parser_context.h

```diff
@@ -20,20 +20,8 @@ namespace Carbon {
 
 // This parser uses a stack for state transitions. See parser_state.def for
 // state documentation.
```

Comment:
I think the comment needs updating to reflect the new structure.

Additionally, if there is any rough guidelines or ideas around which kind of methods should  be here vs. the other constructs, I'd find those both useful when reading and for future changes.

Reply:
> I think the comment needs updating to reflect the new structure.

I'm not clear how much of a change you're looking for -- does this seem okay?

> Additionally, if there is any rough guidelines or ideas around which kind of methods should be here vs. the other constructs, I'd find those both useful when reading and for future changes.

The standard right now is very small, just shared functionality, i.e. things that couldn't go into one of the individual files.